subtests/docker_cli/version/version.py

```diff
@@ -42,12 +41,6 @@ def postprocess(self):
         # Raise exception on Go Panic or usage help message
         outputgood = OutputGood(self.stuff['cmdresult'])
         docker_version = DockerVersion(outputgood.stdout_strip)
-        info = ("docker version client: %s server %s"
-                % (docker_version.client, docker_version.server))
-        self.loginfo("Found %s", info)
```

Comment:
Fine to strip out the file-writing, but no harm in leaving the ``loginfo()`` in there, no?  If anything it continuously verifies the ``docker_version`` class is working.  It's failed one time that I remember, when docker's output format changed.